pkgs/development/libraries/libelf/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv
-, fetchurl, autoreconfHook, gettext
+, fetchurl, autoreconfHook, gettext, netbsd
 }:
 
 # Note: this package is used for bootstrapping fetchurl, and thus
```

Comment:
I wonder, can we use fetchcvs if we are to bootstrap fetchurl (natively) on NetBSD later? In any case, this doesn't matter presently at least…

Reply:
Yeah.  I would still like to switch to `fetchurl` with the NetBSD release tarballs instead of CVS.  Don't remember what stopped me last time I looked into it.